tutorials/Item_Response_Theory.ipynb

```diff
@@ -3775,8 +3778,22 @@
   }
  ],
  "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3 (ipykernel)",
+   "language": "python",
+   "name": "python3"
+  },
   "language_info": {
-   "name": "python"
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.7.4"
```

Comment:
It seems like your jupyter kernel is filling this metadata after you run the notebook :). Do you mind cleaning them by running `make format-notebook nb=tutorials/Item_Response_Theory.ipynb` from the root directory of this repo?

(We originally written this script because our internal notebook runtime inserts a ton of irrelevant metadata on each notebook, and we haven't decided what to do with the more general jupyter metadata yet. So at least for now it will be great if we can always keep the metadata clean to avoid changing this back and forth between commits)

Reply:
Done!